content/blog/foundational-software/index.md

```diff
@@ -0,0 +1,221 @@
+
+Just because we focus on foundational software doesn't mean we can't do other things.
+But the focus is to make sure that Rust stays true to its roots.
+
+## Systems That You Plan To Maintain For Years To Come
+
+So, what *is* foundational software?
+
+It's software that organizations deem critical for their success. 
+It might be:
+
+- a robust web backend
+- a package manager
+- a platform for robotics
+- a storage layer
+- a satellite control system 
+
+and many, many more.
+
+All of these things power organizations and *must not fail* or at least do so *gracefully*.
+My clients and the companies listed on our [podcast page](/podcast) all have one thing in common: 
+They work on Rust projects that are not on the sideline, but front and center, and they shape the future of their infrastructure.
+
+Rust is useful in situations where the ["worse is better" philosophy falls apart](https://dreamsongs.com/Files/worse-is-worse.pdf): it's a language for building the "right thing":
+
+> With the right thing, designers are equally concerned with simplicity, correctness, consistency, and completeness.
+
+I think many companies will choose Rust to build their future platforms on.
+As such, it competes with C++ as much as it does with Kotlin or Python.
+
+I believe that we should shift the focus away from memory safety (which these languages also have) and instead focus on the explicitness, expressiveness, and ecosystem of Rust that is highly competitive with these languages.
+It is a language for teams which want to build things right and are at odds with the "move fast and break things" philosophy of the past.
+Rust is future-looking. 
+Backwards-compatibility is enforced by the compiler and many people work on the robustness aspect of the language.
+
+Dropbox was one of the first production users of Rust.
+They built their storage layer on top of it.
+At no point did they think about using Rust as a C++ replacement.
+Instead, they saw the potential of Rust as a language for building scalable and reliable systems.
+Many more companies followed: 
+Amazon, Google, Microsoft, Meta, Discord, Cloudflare, and many more.
+These organizations build platforms.
+Rust is a tool for professional programmers, developed by world experts over more than a decade of hard work.
+
+## Rust Is A Tool For Professionals
+
+Is Rust used for real?
+
+Graydon Hoare says:
+
+> "At this point, we now know the answer: yes, Rust is used a lot. It's used for real, critical projects to do actual work by some of the largest companies in our industry. We did good."
+>
+> "[Rust is] not a great hobby language but it is a fantastic professional language, precisely because of the ease of refactors and speed of development that comes with the type system and borrow checker."
+> 
+> -- [10 Years of Stable Rust: An Infrastructure Story](https://rustfoundation.org/media/10-years-of-stable-rust-an-infrastructure-story/)
```

Comment:
```suggestion
> "At this point, we now know the answer: yes, Rust is used a lot. It's used for real, critical projects to do actual work by some of the largest companies in our industry. We did good."
>
> "[Rust is] not a great hobby language but it is a fantastic professional language, precisely because of the ease of refactors and speed of development that comes with the type system and borrow checker."

-- Graydon Hoare, [10 Years of Stable Rust: An Infrastructure Story](https://rustfoundation.org/media/10-years-of-stable-rust-an-infrastructure-story/)
```